### PR TITLE
YWOE 6 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/loch_larent.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Name:Loch Larent
+ManaCost:no cost
+Types:Land
+K:CARDNAME enters the battlefield tapped.
+A:AB$ Mana | Cost$ T | Produced$ U | SpellDescription$ Add {U}.
+A:AB$ Scry | Cost$ 1 U T | ScryNum$ 3 | GameActivationLimit$ 1 | PlayerTurn$ True | SubAbility$ DBEffect | SpellDescription$ Scry 3.Target opponent gets a one-time boon with "When you cast a creature spell, that creature enters the battlefield tapped and with a stun counter on it." Activate only during your turn and only once.
+SVar:DBEffect:DB$ Effect | Boon$ True | Duration$ Permanent | Triggers$ SpellCast | ValidTgts$ Opponent | EffectOwner$ Targeted
+SVar:SpellCast:Mode$ SpellCast | ValidCard$ Creature | ValidActivatingPlayer$ You | OneOff$ True | ActivationLimit$ 1 | TriggerZones$ Command | Execute$ TrigEffect | TriggerDescription$ When you cast a creature spell, that creature enters the battlefield tapped and with a stun counter on it.
+SVar:TrigEffect:DB$ Effect | ReplacementEffects$ ETBAddCounter,RETBTapped | RememberObjects$ TriggeredCard
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/loch_larent.txt:9]: single RE + effect needs to clean itself up?

File: forge-gui/res/cardsfolder/upcoming/stormkeld_curator_giant_secrets.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+Name:Stormkeld Curator
+ManaCost:4 W
+Types:Creature Giant
+PT:6/6
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigAurify | TriggerDescription$ When CARDNAME enters the battlefield, you may put any number of Aura cards from your graveyard and/or hand onto the battlefield attached to it.
+SVar:TrigAurify:DB$ RepeatEach | RepeatSubAbility$ BrunaAttach | RepeatCards$ Aura.CanEnchantSource+NotAttachedTo | SubAbility$ HandAuras
+SVar:BrunaAttach:DB$ Attach | Object$ Remembered | Defined$ Self | Optional$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/stormkeld_curator_giant_secrets.txt:7]: bad copy pasta

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/stormkeld_curator_giant_secrets.txt:7]: yeah I fixed that in #3918 – want to just pull this card @Simisays ?

File: forge-gui/res/cardsfolder/upcoming/syr_joshua_and_syr_saxon.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Syr Joshua and Syr Saxon
+ManaCost:G W
+Types:Legendary Creature Human Knight
+PT:2/3
+K:Double team
+S:Mode$ IgnoreLegendRule | ValidCard$ Permanent.namedSyr Joshua and Syr Saxon | IsPresent$ Permanent.namedSyr Joshua and Syr Saxon | PresentCompare$ EQ2 | Description$ If you control exactly two legendary permanents named Syr Joshua and Syr Saxon, the "legend rule" doesn't apply to them.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/syr_joshua_and_syr_saxon.txt:6]: Hmm, I think the IsPresent should contain `Legendary`

There is potential where you might have two "Syr J and Syr S" but one isn't legendary... Double Major for example

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/syr_joshua_and_syr_saxon.txt:6]: Oh good catch. It even says "legendary permanents named" in the oracle

File: forge-gui/res/cardsfolder/upcoming/syr_joshua_and_syr_saxon.txt [hunk 4]
```diff
@@ -4,5 +4,5 @@ Types:Legendary Creature Human Knight
 PT:2/3
 K:Double team
 S:Mode$ IgnoreLegendRule | ValidCard$ Permanent.namedSyr Joshua and Syr Saxon | IsPresent$ Permanent.namedSyr Joshua and Syr Saxon | PresentCompare$ EQ2 | Description$ If you control exactly two legendary permanents named Syr Joshua and Syr Saxon, the "legend rule" doesn't apply to them.
-S:Mode$ Continuous | Affected$ Creature.YouCtrl+Other+namedSyr Joshua and Syr Saxon | AddKeyword$ Battle Cry | Description$ Each other creature you control named Syr Joshua and Syr Saxon has battle cry.
+S:Mode$ Continuous | Affected$ Creature.YouCtrl+Legendary+Other+namedSyr Joshua and Syr Saxon | AddKeyword$ Battle Cry | Description$ Each other creature you control named Syr Joshua and Syr Saxon has battle cry.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/syr_joshua_and_syr_saxon.txt:7]: Oops not this line... line 6!

**Review by Northmoc — APPROVED** [review]
Looks good to me,

**Pull request merged**